docs/user/container.md

```diff
@@ -315,7 +418,27 @@ Server Checksumming   on
     Note that currently, once a container is created, its checksum configuration
     cannot be changed.
 
-### Inline Deduplication (Preview)
+### Erasure Code
+
+DAOS erasure code implementation uses a fixed cell size that applies to all
+objects in the container. The cell size in DAOS is the size of each data and
+parity fragments (also called sometimes chunks). The cell size can be set at
+container creation time via the property:
+
+```bash
+$ daos cont create tank --label mycont5 --type POSIX --properties rf:1,cell_size:65600
+  Container UUID : 90185799-0e22-4a0b-be9d-1a20900a35ee
```

Comment:
64kiB is 65536, why does this say 65600?  does it support saying 64k or 1M or ... using a suffix?

Reply:
Fixed. It sounds like we don't support k/m suffix. I will create a story for this.